### PR TITLE
Added scripts for ci infrastructure

### LINKED ISSUE #418: Add continues integration automation scripts
## Description
As a developer, my PR's should be tested automatically delivering appropriate reports in multiples formats (HTML including graphs, tables, images, XML etc). This will enable this will permit the project to be easily setup for continues deployment.

## Acceptance Criteria
- [x] Add test `.sh` scripts (that will be used in jenkins' jobs) 
- [x] Modify documentation (AUTOMATION.md) to show how setup is done.

## Definition of Done
- [x] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
5 hours

### PR DESCRIPTION
**Pull request by Fenn-CS**
### Description
Added scripts that will be supplied to jenkins for test jobs

Fixes #418 

### Type of Change:
- Code
- Quality Assurance
- Documentation

### How Has This Been Tested?
Tested locally

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by nida** [comment]
What has failed?

**Comment by Fenn-CS** [comment]
@nida it's flake8 I just need to fix formatting

File: systers_portal/systers_portal/settings/runner.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        """Run pytest and return the exitcode.
+
+        It translates some of Django's test command option to pytest's.
+        """
+        import pytest
+
+        argv = []
+        if self.verbosity == 0:
+            argv.append('--quiet')
+        if self.verbosity == 2:
```
Comment (nida) [inline systers_portal/systers_portal/settings/runner.py]: Out of curiosity, what would verbosity=1 mean?

Comment (Fenn-CS) [reply systers_portal/systers_portal/settings/runner.py]: Hello @nida by default the value is one. 
https://github.com/Fenn-CS/portal/blob/5f43573a75e81075906f83b17df76ef6b887f34a/systers_portal/systers_portal/settings/runner.py#L4

And on one, all the "verbose" reports irrespective will be displayed on shell.

File: scripts/run_tests.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+# source venv/bin/activate
+
+#Install Requirements
+
+# pip install -r requirements.txt
+
+#Run tests, $1, $2, $3 ... allow for command line argurments
+
+python systers_portal/manage.py test --settings=systers_portal.settings.testing $1 $2 $3 $4 $5
```
Comment (nida) [inline scripts/run_tests.sh]: What are these parameters $1 you provide description in the comment.

File: systers_portal/common/tests/selenium/base.py [hunk 3]
```diff
@@ -28,7 +28,7 @@ class SeleniumTestCase(StaticLiveServerTestCase):
     def setUpClass(cls):
         super().setUpClass()
         cls.browser = browsers['firefox']()
-        cls.browser.implicitly_wait(0)
+        cls.browser.implicitly_wait(30)
```
Comment (nida) [inline systers_portal/common/tests/selenium/base.py]: Can you add a comment about why the time is set to 30s

File: systers_portal/systers_portal/settings/runner.py (hunk 1 above)
Comment (nida) [reply systers_portal/systers_portal/settings/runner.py]: Makes sense. Thanks for the explanation

File: systers_portal/common/tests/selenium/base.py (hunk 3 above)
Comment (nida) [reply systers_portal/common/tests/selenium/base.py]: Comment in the file.

**Comment by Fenn-CS** [comment]
This PR is not in progress and has not been abandoned either, some parts of it (Jenkins scripts) have to be removed while the changes regarding the pytest runner maintained.

**Comment by sakshi1499** [comment]
@sammy1997 I don't think this PR can be merged. There are 5 commits and tests have failed too. Should I close it then?

**Comment by sakshi1499** [comment]
Closing this PR as there isn't any response, tests are failing and changes cannot be merged.

**Pull request closed without merging**